games-emulation/jrommanager/jrommanager-1.6-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=JRomManager
+MY_PV=1.6b20
+MY_P="${MY_PN}-${MY_PV}"
+
+inherit desktop
+
+DESCRIPTION="A Mame and Retrogaming Rom Manager, Java alternative to ClrMamePro"
+HOMEPAGE="https://github.com/optyfr/JRomManager"
+SRC_URI="https://github.com/optyfr/${MY_PN}/releases/download/${MY_PV}/${MY_P}.zip -> ${P}.zip"
```

Comment:
Why are you changing the filename here? The original filename doesn't seem to be wrong in any way.

Reply:
well... Because Jonas Stein told me to : "rename the SRC_URI with -> ${P}"  
(see https://bugs.gentoo.org/660900#c1)

Reply:
He probably missed that upstream includes full MY_P in the URI. We require renames when the original filename contains version only.